exampleCourse/courseInstances/SectionA/assessments/03-LectureContent/.ipynb_checkpoints/Creating_plot-checkpoint.ipynb

```diff
@@ -0,0 +1,246 @@
+{
```

Comment:
Can we get rid of these checkpoint files (in `.ipynb_checkpoints` directories)?

Reply:
sure, I will delete them.